ang/test/civicase-base/directives/checkbox.directive.spec.js

```diff
@@ -0,0 +1,231 @@
+/* eslint-env jasmine */
+
+(() => {
+  describe('checkbox', () => {
+    let $controller, $scope, event;
+    const trueValue = 'this is correct';
+    const falseValue = 'this is not correct';
```

Comment:
Can we set these two variables inside the describe block they are used in?

Reply:
They are used in multiple blocks. These are defined at the top to avoid repeating them in multiple blocks.